nengo/utils/graphs.py

```diff
@@ -98,6 +98,43 @@ def toposort(edges):
     return ordered
 
 
+def transitive_closure(edges, topo_sorted=None):
+    """Constructs the transitive_closure of a DAG.
```

Comment:
Cool. Then I think it would be a good idea to include the synonym "list of adjacencies" in the description.